jcg_testcases/src/main/scala/TestCaseExtractor.scala

```diff
@@ -18,6 +18,8 @@ trait TestCaseExtractor {
       """(?<body>.*?)""" + // multiple code snippets
       """\[//\]: \# \(END\)""").r // [//]: # (END)
 
+    protected val re: Regex
```

Comment:
I think this could have a slightly more specific name to document what it is used for.